src/app/globals.css

```diff
@@ -17,11 +17,67 @@
 }
 
 body {
+  font-family: 'Courier Prime';
   color: rgb(var(--foreground-rgb));
-  background: linear-gradient(
+  /* background: linear-gradient(
       to bottom,
       transparent,
       rgb(var(--background-end-rgb))
     )
-    rgb(var(--background-start-rgb));
+    rgb(var(--background-start-rgb)); */
+    background-color: black;
 }
+
+.noise-gradient-container {
+  background: url('/Group.svg') center center;
+  background-size: cover;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: -1;
+}
+
+
+@font-face {
+  font-family: 'Courier Prime';
+  src: url('/fonts/CourierPrime-Regular.ttf') format('truetype');
+}
+
+
+
+.heroTitle {
+  @apply text-white text-5xl tracking-wide ;
+  line-height: 80px;
+}
+
+.red-600{
+  color:#AC0000;
+}
+.heroImageContainer {
+  @apply xl:flex-[1.5] flex justify-end items-end w-full xl:h-screen;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items:center;
+  width: 100%;
+  height: 100vh;
+  background-blend-mode: difference;
+}
+
+.heroImage {
+  @apply  xl:w-full w-[90%] xl:h-full h-[590px] ;
+  background-image: url('/hero_streetimage.png');
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  background-blend-mode: difference;
+  /* opacity: 0.75; */
+}
```

Comment:
If you need these hero selectors only for the Hero component, don't put it in global styling file

Reply:
The global css, basically didn't initially include './components/**/*.{js,ts,jsx,tsx,mdx}' so tailwind didn't apply to my hero component which i created first. Didn't realize that that was the reason (kinda got annoyed because of it :) ) and I didn't include it so that was a quick fix of mine (i suppose, obviously not a good one). Forgot to go back later and delete it when i add the line to the Tailwind CSS configuration and got it working.  Thanks on the remark though, gonna fix it now.